From 0c2ac7f964413b451df1bb824d70bacb6207ae27 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sun, 23 Feb 2020 18:56:41 +0200
Subject: [PATCH 17/17] Revert "add Pixel accent overlay"

This reverts commit 6322d16ea92066aadcdb7ca604a04f826419d113.

	deleted:    packages/overlays/AccentColorPixelOverlay/Android.mk
	deleted:    packages/overlays/AccentColorPixelOverlay/AndroidManifest.xml
	deleted:    packages/overlays/AccentColorPixelOverlay/res/values/colors_device_defaults.xml
	deleted:    packages/overlays/AccentColorPixelOverlay/res/values/strings.xml
	modified:   packages/overlays/Android.mk
---
 .../AccentColorPixelOverlay/Android.mk        | 31 -------------------
 .../AndroidManifest.xml                       | 25 ---------------
 .../res/values/colors_device_defaults.xml     | 22 -------------
 .../res/values/strings.xml                    | 22 -------------
 packages/overlays/Android.mk                  |  1 -
 5 files changed, 101 deletions(-)
 delete mode 100644 packages/overlays/AccentColorPixelOverlay/Android.mk
 delete mode 100644 packages/overlays/AccentColorPixelOverlay/AndroidManifest.xml
 delete mode 100644 packages/overlays/AccentColorPixelOverlay/res/values/colors_device_defaults.xml
 delete mode 100644 packages/overlays/AccentColorPixelOverlay/res/values/strings.xml

diff --git a/packages/overlays/AccentColorPixelOverlay/Android.mk b/packages/overlays/AccentColorPixelOverlay/Android.mk
deleted file mode 100644
index 28fbfde6..00000000
--- a/packages/overlays/AccentColorPixelOverlay/Android.mk
+++ /dev/null
@@ -1,31 +0,0 @@
-#
-#  Copyright 2019, The Android Open Source Project
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-LOCAL_PATH:= $(call my-dir)
-include $(CLEAR_VARS)
-
-LOCAL_RRO_THEME := AccentColorPixel
-
-LOCAL_PRODUCT_MODULE := true
-
-LOCAL_SRC_FILES := $(call all-subdir-java-files)
-
-LOCAL_RESOURCE_DIR := $(LOCAL_PATH)/res
-
-LOCAL_PACKAGE_NAME := AccentColorPixelOverlay
-LOCAL_SDK_VERSION := current
-
-include $(BUILD_RRO_PACKAGE)
diff --git a/packages/overlays/AccentColorPixelOverlay/AndroidManifest.xml b/packages/overlays/AccentColorPixelOverlay/AndroidManifest.xml
deleted file mode 100644
index c6011e22..00000000
--- a/packages/overlays/AccentColorPixelOverlay/AndroidManifest.xml
+++ /dev/null
@@ -1,25 +0,0 @@
-<!--
-/**
- * Copyright (c) 2019, The Android Open Source Project
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
--->
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.android.theme.color.pixel"
-    android:versionCode="1"
-    android:versionName="1.0">
-    <overlay android:targetPackage="android" android:category="android.theme.customization.accent_color" android:priority="1"/>
-
-    <application android:label="@string/accent_color_pixel_overlay" android:hasCode="false"/>
-</manifest>
diff --git a/packages/overlays/AccentColorPixelOverlay/res/values/colors_device_defaults.xml b/packages/overlays/AccentColorPixelOverlay/res/values/colors_device_defaults.xml
deleted file mode 100644
index 30e4be62..00000000
--- a/packages/overlays/AccentColorPixelOverlay/res/values/colors_device_defaults.xml
+++ /dev/null
@@ -1,22 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!--
-/**
- * Copyright (c) 2019, The Android Open Source Project
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
--->
-<resources>
-    <color name="accent_device_default_light">#1A73E8</color>
-    <color name="accent_device_default_dark">#8AB4F8</color>
-</resources>
diff --git a/packages/overlays/AccentColorPixelOverlay/res/values/strings.xml b/packages/overlays/AccentColorPixelOverlay/res/values/strings.xml
deleted file mode 100644
index 3303cb2d..00000000
--- a/packages/overlays/AccentColorPixelOverlay/res/values/strings.xml
+++ /dev/null
@@ -1,22 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
-/**
- * Copyright (c) 2019, The Android Open Source Project
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
--->
-<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <!-- Pixel accent color name application label. [CHAR LIMIT=50] -->
-    <string name="accent_color_pixel_overlay" translatable="false">Pixel</string>
-</resources>
diff --git a/packages/overlays/Android.mk b/packages/overlays/Android.mk
index 0e80b54a..33fd757e 100644
--- a/packages/overlays/Android.mk
+++ b/packages/overlays/Android.mk
@@ -24,7 +24,6 @@ LOCAL_REQUIRED_MODULES := \
 	AccentColorOrchidOverlay \
 	AccentColorSpaceOverlay \
 	AccentColorGreenOverlay \
-	AccentColorPixelOverlay \
 	AccentColorPurpleOverlay \
 	DisplayCutoutEmulationCornerOverlay \
 	DisplayCutoutEmulationDoubleOverlay \
-- 
2.17.1

